resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,73 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('app.login') }}</div>
+
+                <div class="card-body">
+                    <form method="POST" action="{{ route('login') }}">
+                        @csrf
+
+                        <div class="form-group row">
+                            <label for="email" class="col-md-4 col-form-label text-md-right">{{ __('app.email') }}</label>
```

Comment:
fix hết toàn bộ nhé. các file khác nưaqx

Reply:
chô này tương tự trên em gọi file app.php key 'email' để nó i18n. Em sẽ tìm và fix các chỗ khác.